src/api/database/compact.rst

```diff
@@ -22,15 +22,15 @@
     Request compaction of the specified database. Compaction compresses the
     disk database file by performing the following operations:
 
-    - Writes a new, optimised, version of the database file, removing any
+    - Writes a new, optimized, version of the database file, removing any
       unused sections from the new version during write. Because a new file is
       temporarily created for this purpose, you may require up to twice the
       current storage space of the specified database in order for the
       compaction routine to complete.
-
-    - Removes old revisions of documents from the database, up to the
-      per-database limit specified by the ``_revs_limit`` database
-      parameter.
+    - Removes the bodies of any old, non-leaf revisions of documents from the
```

Comment:
"old" is redundant, "non-leaf" captures the meaning itself. We don't want to imply that wallclock time matters.